en/docs/create-api/create-and-attach-api-policies/api-policies-overview.md

```diff
@@ -17,7 +17,7 @@ Attaching a policy to a specific level is descibed under each policy.
 
 ## Policy Execution Order
 
-When you have multiple policies attahced in different levels policies, you need to aware about in which order they are excuted in runtime. The following table describes the execution order. Top one executed at first and the bottom one exected at last.
+When you have multiple policies attached in different levels policies, you need to aware about in which order they are excuted in runtime. The following table describes the execution order. Top one executed at first and the bottom one exected at last.
```

Comment:
When you have multiple policies attached in different levels policies, you need to **be** aware of in which order they are **executed** **in** runtime. The following table describes the execution order. **The top** one **is executed** at first and the bottom one **is executed** at last.

Reply:
Added content with bold that needs to be changed.

Reply:
Fixed in PRs https://github.com/wso2/docs-apk/pull/583 and https://github.com/wso2/docs-apk/pull/584